docs/introduction-to-smart-contracts.rst

```diff
@@ -84,27 +84,28 @@ registering with username and password — all you need is an Ethereum keypair.
 
     contract Coin {
         // The keyword "public" makes those variables
-        // readable from outside.
+        // easily readable from outside.
         address public minter;
         mapping (address => uint) public balances;
 
         // Events allow light clients to react to
         // changes efficiently.
         event Sent(address from, address to, uint amount);
 
-        // This is the constructor whose code is
+        // This is the so-called constructor whose code is
```

Comment:
Why 'so-called'? Is it the constructor?

Reply:
so-called because this is the first time the term is used.

Reply:
@chriseth Maybe "this method is called the constructor, and its code is…", something like that.

Reply:
I'll just remove it, because I don't want to get into details about whether this is a function or method although it does not use the 'function' keyword.